spec/dummy/app/models/invoice.rb

```diff
@@ -1,17 +1,20 @@
-require 'enumerize'
 require 'aasm'
 
 class Invoice < ActiveRecord::Base
-  extend ::Enumerize
   include AASM
   include ImageUploader::Attachment(:picture)
+  after_initialize :set_default_state, if: :new_record?
 
   belongs_to :category
   belongs_to :city
   has_and_belongs_to_many :items
   has_and_belongs_to_many :other_items, class_name: 'Item'
 
-  enumerize :state, in: [:pending, :rejected, :approved], default: :pending
+  enum state: {
+    pending: 'pending',
+    rejected: 'rejected',
+    approved: 'approved',
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Avoid comma after the last item of a hash.